mixer/test/client/quota_cache/quota_cache_test.go

```diff
@@ -91,6 +89,10 @@ func TestQuotaCache(t *testing.T) {
 	// Check stats for Check, Quota and report calls.
 	if respStats, err := s.WaitForStatsUpdateAndGetStats(2); err == nil {
 		s.VerifyStats(respStats, expectedStats)
+		// Because prefetch code may have some margin, actual number of check and quota calls are not
+		// determined.
+		s.VerifyStatsGE(respStats, "http_mixer_filter.total_remote_check_calls", 2)
+		s.VerifyStatsGE(respStats, "http_mixer_filter.total_remote_quota_calls", 2)
```

Comment:
if not deterministic, one way is 
 we don't need to verify them.  

or it is better to  check < 5